stapi-fastapi/tests/shared.py

```diff
@@ -92,7 +92,7 @@ class OffNadirRange(BaseModel):
     maximum: int = Field(ge=0, le=45)
 
     @model_validator(mode="after")
-    def validate_range(self) -> Self:
+    def validate_range(self) -> "OffNadirRange":
```

Comment:
Can we use `from __future__ import annotations` to be able to return the type (rather than the quoted type)?